windows-calico/limitations.md

```diff
@@ -9,7 +9,7 @@ canonical_url: '/getting-started/windows-calico/limitations'
 | **Feature** | **Not supported**                                            |
 | ----------- | ------------------------------------------------------------ |
 | Install     | Operator install                                             |
-|             | AKS, GKE, IKS, AWS, GCE, Azure, OpenShift, OpenStack, flannel |
+|             | GKE, IKS, OpenStack                                          |
 |             | K3 clusters                                                  |
 |             | Typha component for scaling (Linux-based feature)            |
 |             | ebpf (Linux-based feature)                                   |
```

Comment:
I believe the table has another error per our PR for EE. So not supported should say: Overlay mode with BGP peering. 

@song-jiang 
"Non-overlay mode with BGP peering. That is what we supported. We don't support Overlay mode with BGP peering."